exercises/nucleotide-count/nucleotide_count.ml

```diff
@@ -0,0 +1,10 @@
+open Base
+(* open Base.Continue_or_stop *)
```

Comment:
I'd prefer not to add this, it might be confusing or might lead people to a particular solution when they should find there own way.

Reply:
Good point.